schema.sql

```diff
@@ -55,13 +55,45 @@ CREATE TABLE vets (
 
 -- create specializations table
 CREATE TABLE specializations (
-    vets_id int REFERENCES vets (id),
+    vet_id int REFERENCES vets (id),
     species_id int REFERENCES species (id)
 );
 
 -- Create visits table
 CREATE TABLE visits (
-    animal_id int REFERENCES animals (id),
+    animal_id int REFERENCES animal (id),
     vet_id int REFERENCES vets (id),
     date_of_visit date
 );
+------------------------------------------------------------------
+-- project requirements;
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+-- Use EXPLAIN ANALYZE on the previous queries to check what is happening. Take screenshots of them -
+
+-- before improve
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT COUNT(*) FROM visits where animal_id = 4;
+
+-- after improve
+CREATE INDEX visits_animal_id ON visits(animal_id);
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+-- SELECT * FROM visits where vet_id = 2; Before improve
+SELECT * FROM visits WHERE vet_id = 2;
+explain analyse SELECT COUNT(*) FROM visits where animal_id = 2;
+
+-- Querie : SELECT * FROM visits where vet_id = 2; After improve
+CREATE INDEX visits_vet_id ON visits(vet_id);
+EXPLAIN ANALYSE SELECT * FROM visits WHERE animal_id = 2;
+
+-- Querie :SELECT * FROM owners where email = 'owner_18327@mail.com';
+-- Before
+EXPLAIN ANALYSE SELECT * FROM owners WHERE email ='owner_18327@mail.com';
+SELECT FROM owners WHERE email='owner_18327@mail.com';
+
+-- after
+CREATE INDEX owners_email ON owners(email);
```

Comment:
- [ ] The planning and execution time should be shorter than before creating the index, I suggest using ASC or DESC in the id column to reduce the times for good optimization. 👍🏽